docs/query-language.md

```diff
@@ -341,6 +341,14 @@ have the attribute specified by `field` with one of the values specified in
 
 **Result Type:** array of key/count objects
 
+**Caveats:**
+* TopN on a frame with ranked cache will return the top bitmaps sorted by count.
+* Non-ranked frames maintain a LRU (least recently used) cache, thus a TopN query on this frame will return bitmaps sorted in most recently setbit order.
```

Comment:
Frames with cache type `lru` will maintain an LRU (Least Recently Used) cache, thus a `TopN()` query on this type of frame will return bitmaps sorted in order of most recently set bit.